context/llpcContext.h

```diff
@@ -292,6 +294,8 @@ class Context : public llvm::LLVMContext
         llvm::Type* pDoubleTy;    // Double
         llvm::Type* pVoidTy;      // Void
 
+        llvm::Type* pInt16x2Ty;   // Int16 x 2
+        llvm::Type* pInt16x4Ty;   // Int16 x 4
```

Comment:
It seems this type isn't used, please remove it.

Reply:
commit updated